client/valor/coretypes.py

```diff
@@ -287,7 +287,6 @@ class Annotation:
     metadata = DictionaryProperty("annotation_metadata")
     bounding_box = GeometryProperty("bounding_box")
```

Comment:
Multipolygon is still a valid user input since its converted to the raster type.

The point of this PR was to remove multipolygon from places where we dont use it, such as in the DB.